uwac/libuwac/uwac-input.c

```diff
@@ -673,6 +673,7 @@ static void pointer_handle_axis(void *data, struct wl_pointer *pointer, uint32_t
 	event->value = value;
 }
 
+#if 0
 static void pointer_frame(void *data, struct wl_pointer *wl_pointer)
```

Comment:
This #if 0 is incorrect, if you have warnings it's because the wayland you're compiling against is old (and BTW we could make things better by testing the protocol version)